File: 10gameslistviewdisplayintextview/app/src/main/java/com/example/a10gameslistviewdisplayintextview/MainActivity.java
package com.example.a10gameslistviewdisplayintextview;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private ListView gameListView;
    private TextView selectedGameTextView;
    private String[] games = {"Game 1", "Game 2", "Game 3", "Game 4", "Game 5", "Game 6", "Game 7", "Game 8", "Game 9", "Game 10"};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        gameListView = findViewById(R.id.game_list);
        selectedGameTextView = findViewById(R.id.selected_game);

        ArrayAdapter<String> adapter = new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, games);
        gameListView.setAdapter(adapter);

        gameListView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                String selectedGame = games[position];
                selectedGameTextView.setText(selectedGame);
            }
        });
    }
}

File: ListView/app/src/main/java/com/example/listview/MainActivity.kt
package com.example.listview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Gravity
import android.widget.ArrayAdapter
import android.widget.ListView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val animals= arrayOf<String>("Tiger","Cheetah","Jaguar","Leopard","Lion","Elephant","Bear","Dog","Cat","Otter")
        val listview=findViewById<ListView>(R.id.mylist)
        listview.adapter=ArrayAdapter<String>(this,android.R.layout.simple_list_item_1,animals)
        listview.setOnItemClickListener { adapterView, view, i, l ->
            val toast=Toast.makeText(this,"You clicked on the animal\n"+animals[i],Toast.LENGTH_LONG)
            toast.setGravity(Gravity.CENTER,0,0)
            toast.show()
        }
    }
}